client/app/components/preprint-basics/component.js

```diff
@@ -183,8 +177,17 @@ export default Ember.Component.extend(BASICS_VALIDATIONS, {
     },
     init(){
         this._super(...arguments);
-        this.get('store').findRecord('node', ENV.NODE_GUID).then((result)=>{
+        this.get('store').findRecord('node', ENV.nodeGuid).then((result)=>{
```

Comment:
Sure, just remember to do a find/replace when renaming `ENV.nodeGuid`. I noticed this variable was not changed after `ENV.NODE_GUID` was renamed to `ENV.nodeGuid`and that caused errors.